http/headers/cross-origin-resource-policy.json

```diff
@@ -6,10 +6,12 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/HTTP/Headers/Cross-Origin-Resource-Policy",
           "support": {
             "chrome": {
-              "version_added": "73"
+              "version_added": "73",
+              "notes": "Until version 75, <a href='https://crbug.com/952834'>bug 952834</a> caused downloads to fail"
```

Comment:
```suggestion
              "notes": "Until version 75, downloads for files with this header would fail in Chrome.  See <a href='https://crbug.com/952834'>bug 952834</a>."
```